tests/pybaseball/datahelpers/test_postprocessing.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
+import pandas as pd
+
+import pytest
+
+from pybaseball.datahelpers import postprocessing
+
+
+@pytest.fixture()
+def sample_unprocessed_result():
+    return pd.DataFrame(
+        [
+            ['TBR', '1', '2', '50 %', '8'],
+            ['555', '3', '4', '45%', 'null']
+        ],
+        columns=['Team', 'Runs', 'Hits', 'CS%', 'HR']
+    ).reset_index(drop=True)
+
+
+class TestPostProcessing:
+    def test_coalesce_nulls(self, sample_unprocessed_result):
+        expected_result = pd.DataFrame(
+            [
+                ['TBR', '1', '2', '50 %', '8'],
+                ['555', '3', '4', '45%', np.nan]
+            ],
+            columns=['Team', 'Runs', 'Hits', 'CS%', 'HR']
+        ).reset_index(drop=True)
+
+        actual_result = postprocessing.coalesce_nulls(sample_unprocessed_result).reset_index(drop=True)
+
+        pd.testing.assert_frame_equal(actual_result, expected_result)
```

Comment:
Reviewing the [docs](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.testing.assert_frame_equal.html), should we use `check_dtype=False` to allow for changing the types?

Reply:
We could in some tests (like this one), but would still want it for testing the numeric postprocessor. My opinion is to lean towards specificity, but would be willing to add it if you thought it was appropriate.